docs/status.rst

```diff
@@ -20,6 +20,82 @@ Release Notes
 +++++++++++++
 
 *****
+0.3.6
+*****
+
+This an alpha pre-release that contains some major updates. The most important 
+changes are highlighted in **bold** below. We have also made some updates to 
+the documentation.
+
+Features:
+
+- **Rename package from “ga4gh” to “ga4gh-server”** #1582, #1583
+
+- Added **support for BigWig files** in a new Continuous Data object #1573  New endpoints and message types include:
+ - `POST /continuoussets/search`
```

Comment:
Same thing as on schemas, these backticks aren't doing what you think they might be.